sdk/src/connections/target/adsd3500_sensor.cpp

```diff
@@ -479,9 +481,20 @@ aditof::Status Adsd3500Sensor::setMode(const uint8_t &mode) {
         return status;
     }
 
-    //decide if we read this from ccb or from sdk
-    if (0) {
-        //TO DO: master ccb read from nvm goes here
+    if (m_ccbmEnabled && m_sensorConfiguration == "standard") {
+        bool modeFound = false;
+        for (auto table : m_availableModes) {
+            if (mode == table.modeNumber) {
+                modeTable = table;
+                modeFound = true;
```

Comment:
if table is found then the for can be interrupt with break; statement.

Reply:
Fixed. Thanks, I missed that.